force-app/main/default/classes/UTIL_Describe.cls

```diff
@@ -534,12 +534,23 @@ public class UTIL_Describe {
     }
 
     /*******************************************************************************************************
-    * @description utility to return a list of strings from a field set.
+    * @description utility to return a list of field names from a field set.
     * @param obj the namespaced name of the object
     * @param fieldSet the namespaced name of the object's field set
     * @return a list of strings of field names in the field set
     */
     public static List<String> listStrFromFieldSet(String obj, String fieldSetName) {
+        return listStrFromFieldSet(obj, fieldSetName, false);
+    }
+
+    /*******************************************************************************************************
+    * @description utility to return a list of field names from a field set.
+    * @param obj the namespaced name of the object
+    * @param fieldSet the namespaced name of the object's field set
+    * @param onlyRequired if true, only return required fields
+    * @return a list of strings of field names in the field set
+    */
+    public static List<String> listStrFromFieldSet(String obj, String fieldSetName, Boolean onlyRequired) {
```

Comment:
@lparrott I know this was not a part of your changes, but should we use a label for the error string in this method?  Might be a challenge to construct appropriately in all languages...

Reply:
Good idea, can't hurt to move that to a Label, it will look cleaner in the code too.